src/__tests__/integration/data/staking/erc-20.js

```diff
@@ -73,6 +73,10 @@ export function handle(state, action) {
     const recipient = _input.recipient;
     const amount = _input.amount;
 
+    if (amount == 0 ) {
+      throw new ContractError('TransferFromZero');
```

Comment:
Is amount the sender balance (transferFROMzero suggests that)? Shouldn't we check whether `amount <= 0`?

Reply:
In a real contract - yes.

This contract is a simplified, stripped-down version (I believe there's a warn in the header) used only for testing certain features (like this specific case, where a transfer from zero was an indication of some issue in the internal writes feature in the whole test scenario)